package/MDAnalysis/lib/pkdtree.py

```diff
@@ -145,75 +146,87 @@ def set_coords(self, coords):
         self.kdt.set_data(wrapped_data)
         self.built = 1
 
-    def find_centers(self, center_point, radius):
+    def find_images(self, center_points, radius):
         """
-        Find relevant images of a center point, inspired by
+        Find relevant images of a a set of center points, inspired by
         https://github.com/patvarilly/periodic_kdtree
 
         Parameters
         ----------
-        center_point: NumPy.array
-          Coordinates of the query center point
+        center_points: NumPy.array shape=(N,3)
```

Comment:
can you change this to accept an `array_like`? In the function you have to convert center_points to a np.array with `np.asarray(center_points)`. This makes the method more flexible